ocaml/xapi-idl/lib/xcp_service.ml

```diff
@@ -236,6 +255,11 @@ let common_options =
     , (fun () -> !config_dir)
     , "Location of directory containing configuration file fragments"
     )
+  ; ( "timeslice"
+    , Arg.Set_float timeslice
+    , (fun () -> Printf.sprintf "%.3f" !timeslice)
```

Comment:
Not sure seconds is the best unit here when we mostly talk about milliseconds. However, this has enough resolution down to 1ms.